src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs

```diff
@@ -228,21 +228,21 @@ private void InitializeComponent()
             panel1.BackColor = SystemColors.Control;
             panel1.Controls.Add(label6);
             panel1.Dock = DockStyle.Top;
-            panel1.Location = new Point(0, 279);
+            panel1.Location = new Point(0, 304);
+            panel1.MinimumSize = new Size(0, 18);
             panel1.Name = "panel1";
             panel1.Padding = new Padding(0, 0, 0, 3);
-            panel1.Size = new Size(978, 15);
+            panel1.Size = new Size(1324, 18);
             panel1.TabIndex = 7;
             // 
             // label6
             // 
-            label6.Anchor = AnchorStyles.Left;
             label6.AutoSize = true;
             label6.ImeMode = ImeMode.NoControl;
-            label6.Location = new Point(206, -1);
+            label6.Location = new Point(0, 0);
```

Comment:
Links label was probably intended to be above URI column (before there was a header?)
It does not add anything, remove?

Reply:
It does add some clarification esp. when starting fresh
![image](https://github.com/user-attachments/assets/6eaba402-d189-46e8-bb51-48ceb3acf804)

Reply:
I do not consider the label to add anything but am OK with keeping it